### PR TITLE
Adding New-CsOnlineSession

### PR DESCRIPTION
**Pull request by get-itips**
Initial version directly from the output given by PlatyPS

@tomkau please kindly review
@kenwith please kindly review/add any reviewer you consider

Related issue https://github.com/MicrosoftDocs/office-docs-powershell/issues/5160

### CODE REVIEW

**Comment by tseward** [comment]
@get-itips are there no examples of Modern Auth authentication scenarios with this cmdlet?

File: skype/skype-ps/skype/New-CsOnlineSession.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+### OAuthAccessToken
+```
+New-CsOnlineSession [-OAuthAccessToken <SecureString>] [-OverrideAdminDomain <String>]
+ [-OverrideDiscoveryUri <Uri>] [-OverridePowerShellUri <Uri>] [-OverrideAccessTokenResourceUri <Uri>]
+ [-SessionOption <PSSessionOption>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Creates a remote session to Microsoft Skype For Business Online DataCenter.
+In this session, tenant administrator can run Skype For Business cmdlets to manage users, policies and configurations.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsOnlineSession.md]:
```suggestion
In this session, Skype for Business Online administrator can run Skype For Business cmdlets to manage users, policies and configurations.
```

File: skype/skype-ps/skype/New-CsOnlineSession.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Credential
+Specifies a user account that is Tenant Administrator, or Syndicated Partner Administrator.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsOnlineSession.md]:
```suggestion
Specifies a Skype for Business Online administrator, or Syndicated Partner administrator account.
```

File: skype/skype-ps/skype/New-CsOnlineSession.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+Type a user name, such as "User@Domain.com", or enter a PSCredential object, such as one returned by the Get-Credential cmdlet.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsOnlineSession.md]:
```suggestion
Type a Skype for Business Online administrator account name, such as "User@Domain.com", or enter a PSCredential object, such as one returned by the Get-Credential cmdlet.
```

File: skype/skype-ps/skype/New-CsOnlineSession.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+When you type a user name, you will be prompted for a password.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsOnlineSession.md]:
```suggestion
When you type an account name, you will be prompted for a password.
```

File: skype/skype-ps/skype/New-CsOnlineSession.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -OAuthAccessToken
+Specifies access token acquired already by the Tenant Administrator, or Syndicated Partner Administrator.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsOnlineSession.md]:
```suggestion
Specifies access token acquired already by the Skype for Business Online administrator, or Syndicated Partner Administrator.
```

File: skype/skype-ps/skype/New-CsOnlineSession.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -OverrideAdminDomain
+Specifies a domain to be managed.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsOnlineSession.md]:
```suggestion
Specifies a default domain to be managed.
```

File: skype/skype-ps/skype/New-CsOnlineSession.md [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+This could be home domain of signed in user, or a different one that he/she has permission to manage.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsOnlineSession.md]:
```suggestion
```

**Comment by dariomws** [comment]
There is some complicated terminology in this article.

As "Skype for Business Online Administrator" should be "Teams Service Administrator", or maybe even better "Teams administrator".

**Comment by get-itips** [comment]
> There is some complicated terminology in this article.
> 
> As "Skype for Business Online Administrator" should be "Teams Service Administrator", or maybe even better "Teams administrator".

A lot of cmdlets still refer to the admin role as _Skype for Business Online Administrator_ so I would vote to keep it until a major request comes from the Product Group.

**Comment by get-itips** [comment]
> @get-itips are there no examples of Modern Auth authentication scenarios with this cmdlet?

Added another Example to refer to MFA.

**Comment by get-itips** [comment]
Suggested changes were made, please review again as requested.

**Pull request merged**